tests/fail/weak_memory/cpp20_rwc_syncs.rs

```diff
@@ -1,5 +1,14 @@
 // ignore-windows: Concurrency on Windows is not supported yet.
 // compile-flags: -Zmiri-ignore-leaks
+// error-pattern:
+
+// FIXME: This test should be under pass with a [known-bug header]
+// (https://rustc-dev-guide.rust-lang.org/tests/ui.html#known-bugs),
+// but the standalone compiletest-rs does not support this.
```

Comment:
Miri does not use compiletest-rs any more.

But anyway if we are currently testing this to fail, it should be under `fail` (or `panic`) IMO. :shrug: 
Maybe put it into `fail/should-pass` or so to document the fact that we consider this failure a bug?